### PR TITLE
Add Discord invitation on the website, `README.md`, and `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add Discord [invitation](https://discord.gg/hScy8KWACQ) on the website, `README.md`, and `CONTRIBUTING.md`.
  - I added a new SVG for the Discord [logo](https://discord.com/branding), but the `viewBox` (and probably other stuff) are not correct as per the contributing guidelines. Its width and height are modified in `style.css`, though. Maybe the bot should ignore this SVG.

### CODE REVIEW

File: docs/assets/css/style.css [hunk 1]
```diff
@@ -8,6 +8,14 @@ html {
 *, *::after, *::before {
   box-sizing: inherit; }
 
+.discord-logo:before {
```
Comment (Thomas-Boi) [inline docs/assets/css/style.css:11]: This one is my bad. Should be 
`.discord-logo::before {`

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Just need to change the `:before` CSS to `::before`.

File: docs/assets/css/style.css (hunk 1 above)
Comment (Panquesito7) [reply docs/assets/css/style.css:11]: Are you sure? Other instances in this file use `:before`.

Comment (Thomas-Boi) [reply docs/assets/css/style.css:11]: Hmm, I double checked. Seems like both are fine. See second answer of [this](https://stackoverflow.com/questions/7327326/what-is-the-difference-between-before-and-before)

**Review by Thomas-Boi — APPROVED** [review]

**Comment by Thomas-Boi** [comment]
Well, since this svg is not the `icons/` folder, the bot should have no problem with it. Besides that, I'm cool with this PR. I'll wait for amacado to ensure the styling of the Discord icon is good before I merge this.

Good job 👍

**Review by amacado — CHANGES_REQUESTED** [review]
Regarding [this discussion](https://github.com/devicons/devicon/discussions/537) we should point out, that the discord server is unoffical help/support server and the project is still beeing managed via github (issues).

**Review by amacado — APPROVED** [review]

**Review by Thomas-Boi — APPROVED** [review]
This looks good to me 👍

**Pull request merged**